HelpSource/Help.schelp

```diff
@@ -2,38 +2,266 @@ TITLE::Help
 summary::Documentation home
 categories::Help
 
-SuperCollider is an audio server, programming language, and IDE for sound synthesis and algorithmic composition.
-
 NOTE:: link::Guides/News-3_13##News in SuperCollider version 3.13:: ::
 
-SECTION::Search and browse
+
+SECTION:: An introductory overview
+
+SuperCollider* is a cross-platform** environment for audio synthesis and algorithmic composition used by musicians, artists and researchers working with sound, with three main components:
```

Comment:
```suggestion
*SuperCollider* is a cross-platform environment for audio synthesis and algorithmic composition used by musicians, artists and researchers working with sound, with three main components:
```

Reply:
Thanks! However, I have no experience with this kind of work. 
 
I pressed "commit suggestion", and then it seemed that your suggestion was not applied. So I pressed "Resolve conversation". 

What should I do to apply your suggestion? It is not easy to understand the documentation on GitHub.

Reply:
Or does someone else apply the suggestion? Sorry for these questions.. (:

Reply:
@capital-G 

Thank you very much! However, I have no experience with this type of work.
I clicked on 'Commit suggestion' and then it seemed that your suggestion was not being applied. So I clicked on 'Resolve conversation'.

What should I do to get your suggestion committed? The documentation on GitHub is not easy to understand.
Or is someone else committing it?

I will look into how the process works. If your suggestion is ignored, it is my fault. Please let me know if there is anything I can do to get it committed.

Reply:
This was new for me as well.
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request

Reply:
@mtmccrea 
Thanks!

@capital-G 
I think you mean italic style by adding two * symbols around SuperCollider? I changed it to `emphasis::SuperCollider::`. Please let me know if I am wrong.

Yes, updating scdoc.css might be better. I will try again with minimal changes.

Reply:
> I think you mean italic

There may have been a misunderstanding of the original intention of the asterisks being for markdown (bold), when actually those were meant as footnote indicators.